apps/site/pages/en/blog/announcements/making-nodejs-downloads-reliable.md

```diff
@@ -0,0 +1,194 @@
+---
+date: '2024-12-28T12:00:00.000Z'
+category: announcements
+title: Making Node.js Downloads Reliable
+layout: blog-post
+author: flakey5
+---
+
+Last year, we shared [the details behind Node.js' brand new website](https://nodejs.org/en/blog/announcements/diving-into-the-nodejs-website-redesign).
+Today we're back, talking about the new infrastructure serving Node.js' release assets.
+
+<!-- I don't really like this paragraph -->
+
+This blog post goes into the nitty-gritty details on the web infrastructure behind Node.js, its history, and where it stands today.
+It also goes into specifics on what we had in mind and were prioritizing with this overhaul to our infrastructure.
+
+<!-- I feel like there should be more intro here -->
+
+## Some Brief History
+
+At the start of the project in 2009, Node.js release assets (binaries, documentation) were stored on a publicly accessible S3 bucket.
+Around May of 2010, this was changed to be a VPS that hosted the release assets and the Node.js website.
+
+<!-- this is awkward -->
+
+When the io.js fork happened in 2014, a VPS was also used.
+
+After the Node.js and io.js merge in 2015, io.js' VPS (which will be referred to as the origin server from hereon) was repurposed to host both io.js and Node.js releases along with the Node.js website, and remained that way up until recently.
+
+The architecture looked like this:
+
+![A diagram of the old infrastructure's diagram. Cloudflare is used as a cache and NGINX is used for serving static files.](/static/images/blog/announcements/old-release-asset-infra.png)
+
+## Growing Pains
+
+Nowadays, the nodejs.org domain sees over 3 billion requests and 2+ petabytes of traffic per month, with the majority of that going towards release assets.
+
+<!-- cf dash screenshots? -->
+
+This averages to about 1,157 requests per second, with an average bandwidth of 771 mb per second.
+
+<details>
+    <summary>Math</summary>
+<!-- todo double check w/ up to date numbers -->
```

Comment:
need to confirm these numbers

Reply:
also the website appears to not show this anyways, is there a preferable way to display this or should we even have it?

Reply:
cc @ovflowd

Reply:
wdym?